lib/main_common.pm

```diff
@@ -2673,6 +2673,9 @@ sub load_ha_cluster_tests {
     # Show HA cluster status *after* fencing test
     loadtest 'ha/check_after_reboot';
 
+    # Remove all the resources except stonith/sbd
+    loadtest 'ha/remove_rsc';
```

Comment:
Do we want to remove all resources every time we test the HA stack?

Reply:
Yes, this is my idea but I can add this test with only one cluster if you prefer.
@ldevulder WDYT?

Reply:
Yes, I think it would be more flexible if we only remove resources based on a environment variable, so we can add this test in some of the clusters but not in all.

Reply:
Done